tests_e2e/orchestrator/lib/update_arm_template_hook.py

```diff
@@ -50,9 +50,6 @@ def azure_update_arm_template(self, template: Any, environment: Environment) ->
         #
         allow_ssh: str = vm_tags.get("allow_ssh")
         network_security_rule = NetworkSecurityRule(template, is_lisa_template=True)
-        # Disabling the default outbound access due to security requirement
```

Comment:
Now LISA disables the default outbound access, so we do not need to do it on our side.